docs/content/specs-defs/specs/shared/_index.md

```diff
@@ -809,6 +809,14 @@ For example all modules should be released using a semantic version that matches
 
 A module **SHOULD** avoid breaking changes, e.g., deprecating inputs vs. removing.
 
+{{< hint type=tip >}}
```

Comment:
Thanks @ReneHezser for adding.

I did have a brief chat with @jtracey93 in the meantime and he made a good suggestion that we should probably add here. The idea was somewhat along the lines of
- Backwards compatibility should only be ensure if you introduce a breaking change but want to do so as part of a minor update. This way, you can change the schema, while existing code would still work
- A major version update should then do away with that compatibility mode

Reply:
The text can always be improved :-)
What do you think about the change?

Reply:
@AlexanderSehr @jtracey93 @ReneHezser  I think we are missing a note for all of this not being applicable before v1.0.0 is released.

As per [SNFR17](https://azure.github.io/Azure-Verified-Modules/specs/shared/#id-snfr17---category-release---semantic-versioning), before v1.0.0, a minor bump will cover for both breaking changes and backward compatible features.
Therefore, in this phase breaking changes are expected. We should still avoid them, but if they are needed we cannot use a major bump to release them, nor we can ensure backward compatibility for all the breaking changes we may introduce. It's a delicate matter, but simply not realistic to commit to.

Reply:
Great comment @eriqua 
I've added this as an info for best visibility.